<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>847</width>
    <height>520</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>821</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="preprocess">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <attribute name="title">
      <string>Preprocess</string>
     </attribute>
     <widget class="QLabel" name="label_start_column_output">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>180</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>start_column_output</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_sheet_output">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>sheet_output</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_start_column_input">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>140</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>start_column_input</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_filename">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>20</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Filename</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_sheet_input">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>sheet_input</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_save_arg">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>430</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Save arguments</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_filename">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sheet_input">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>60</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sheet_output">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>100</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_start_column_input">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>140</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_start_column_output">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>180</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_browse_file">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>20</y>
        <width>81</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Browse file</string>
      </property>
      <property name="autoRepeatDelay">
       <number>301</number>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_results">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>331</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_run">
      <property name="geometry">
       <rect>
        <x>690</x>
        <y>430</y>
        <width>101</width>
        <height>24</height>
       </rect>
      </property>
      <property name="text">
       <string>Run</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_outlier_detector">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>220</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_outlier_detector">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>220</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>outlier_detector</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_outlier_detector_param">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>260</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_outlier_detector_param">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>260</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>outlier_detector_param</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_train_size">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>300</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_train_size">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>300</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>train_size</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_num_splits">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>340</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>num_splits</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_num_splits">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>340</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_dataset">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>380</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>dataset</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_dataset">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>380</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="menu2">
     <attribute name="title">
      <string>Menu2</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="menu3">
     <attribute name="title">
      <string>Menu3</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="menu4">
     <attribute name="title">
      <string>Menu4</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
